decisions/log/0028-provide-clarity-on-characterization-factors.md

```diff
@@ -0,0 +1,55 @@
+# 28. Provide Clarity on Characterization Factors (AR5 and AR6)
+
+Date: 2024-01-16
+
+## Status
+
+Proposed
+
+## Context
+
+`characterizationFactors` is a mandatory property of the `CarbonFootprint` data type.
+It's data type is `String` and the value MUST be one of the following (see [4.2.2 Properties #characterizationFactors](https://wbcsd.github.io/tr/data-exchange-protocol/#element-attrdef-carbonfootprint-characterizationfactors)):
+- AR6
+  - for the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (IPCC)
+- AR5
+  - for the Fifth Assessment Report of the IPCC
+
+However, some PCFs might be calculated with inputs that use both AR5 and AR6 as these can come from
+different sources (i.e., different actors in the supply chain). Since the IPCC will keep publishing
+reports, this problem tends to get worse in the future.
+
+In the context of the Methodology Working Group, PACT members have expressed a very strong
+preference for having the possibility of including more than one characterization factor per PCF.
+
+## Decision
+
+1. Change the definition of the `characterizationFactors` from:
+    > The IPCC version of the GWP characterization factors used in the calculation of the PCF (see [=Pathfinder Framework=]   Section 3.2.2). The value MUST be one of the following: [...]
+
+    to (roughly):
+    > **Deprecated**
+    >
+    > The IPCC version of the GWP characterization factors used **in the calculation of the largest relative share of the PCF** (see [=Pathfinder Framework=]   Section 3.2.2). The value MUST be one of the following: [...]
+    >
+    > [...]
+    >
+    > **Advisement: This property is deprecated and only kept to ensure backwards-compatibility. It does not replace the (also mandatory) property `characterizationFactorsSources`.**
+
+    (Please refer to the proposed changes to the technical specification for a more detailed and complete diff.)
+
+2. Add the new mandatory property `characterizationFactorsSources` with the new data type `CharacterizationFactorsSources`, defined as a non-empty `Array` of `CharacterizationFactorsSource`. The new data type `CharacterizationFactorsSource` is defined as an `Object` with the mandatory fields `"name"` and `"version"`, both of which are non-empty `Strings`. The value of the `"name"` field SHOULD be `"IPCC Assessment Report"` and the version SHOULD be either `"5"` or `"6"`.
```

Comment:
Suggestion to reference the https://wbcsd.github.io/data-exchange-protocol/v2/#dt-nonemptystring type instead of:

> both of which are non-empty `Strings`.

Reply:
That's indeed how we propose to type them in the Technical Specifications after this change. Please see:
https://github.com/wbcsd/data-exchange-protocol/pull/46/files#diff-d29e4ea6c333959618be484857d31dde8dfc7a40fba9d381925e80f990cd5c0bR1400